plugins/woocommerce/src/Internal/EmailEditor/templates/WooMailTemplate.php

```diff
@@ -0,0 +1,36 @@
+<?php declare(strict_types = 1);
+
+namespace Automattic\WooCommerce\Internal\EmailEditor\Templates;
+
+class WooMailTemplate {
+
+	public function get_slug(): string {
+		return 'woomailtemplate';
```

Comment:
Perhaps `wooemailtemplate` ?

Reply:
Done in https://github.com/woocommerce/woocommerce/pull/55754/commits/c9a010081c8f3a28d53f4bbe61f996abcbfd784b